app.rb

```diff
@@ -5,6 +5,10 @@
 require './lib/user.rb'
 
 class App < Sinatra::Base
+  configure :production, :staging, :development do
+    enable :logging
+  end
```

Comment:
Same level of detail for all of them?

Do we need log rotation then?

Reply:
These logs end up in CloudWatch.

Good question about Log Rotation, I don't know the answer off the top of my head.
Needs further investigation.